src/optimize.js

```diff
@@ -19,6 +20,8 @@ class Optimize implements Experiment {
     // Trigger an experiment execute is a async operation hence we need
     // to fire an event from experiment to trigger the rendering from Google Optimize.
     // This way we are ensuring that Optimize had time to execute the script in the exeriment.
+    pushToDataLayer({ event: `${this.experimentName}.activate` });
```

Comment:
This doesnt belong in this OSS project - should be called from the parent component when callback is called

Reply:
@jooj123 This is only for experiments through Google Optimize, as the event is required to be sent to GO to run the actual experiment

Reply:
Sorry I was confused - didnt know `pushToDataLayer` was also used for Google Optimize
Can you add docs (maybe just link to Google Optimise setup docs) in the readme
Same with Optimisely i guess

Otherwise pushToDataLayer does nothing